docs/metrics/v3.md

```diff
@@ -1,397 +1,409 @@
 # Metrics Version 3
 
-In metrics version 3, all metrics are available under the endpoint:
+In metrics version 3, all metrics are available under the following base endpoint:
 
 ```
 /minio/metrics/v3
 ```
 
-however, a specific path under this is required.
+To query metrics of a specific type, append the appropriate path to the base endpoint.
+You **must** append a path, as querying the base endpoint alone does not return metrics.
```

Comment:
```suggestion
Querying the base endpoint returns no meaningful data.
```

If we return a 200OK then mod the above to state "returns a `200 OK` and no metrics."

Reply:
If we can get an answer for this, I think we are done.

Reply:
We return an error 404.